tests/integration/components/suspense-component-test.js

```diff
@@ -1,27 +1,27 @@
-import { module, test } from 'qunit';
-import { setupRenderingTest } from 'ember-qunit';
-import { render, waitUntil } from '@ember/test-helpers';
-import hbs from 'htmlbars-inline-precompile';
-import { defer as Defer } from 'rsvp';
-import Service from '@ember/service';
-import sinon from 'sinon';
-
-module('Integration | Component | suspense-component', function(hooks) {
+import { module, test } from "qunit";
```

Comment:
Seems vaguely odd to rewrite all of these quotes to double, maybe you use "prettier on save" and this repo is missing a `.prettierrc.js` file?